src/main/java/com/slcube/shop/business/item/repository/ItemQueryRepositoryImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.slcube.shop.business.item.repository;
+
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import com.slcube.shop.business.item.domain.Item;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+
+
+import java.util.List;
+
+import static com.slcube.shop.business.category.domain.QCategory.*;
+import static com.slcube.shop.business.item.domain.QItem.*;
+
+@RequiredArgsConstructor
+public class ItemQueryRepositoryImpl implements ItemQueryRepository {
+
+    private final JPAQueryFactory query;
+
+    @Override
+    public List<Item> findByCategoryId(Long categoryId, Pageable pageable) {
+
+        return query.selectFrom(item)
+                .join(item.category, category)
+                .where(
+                        category.id.eq(categoryId),
```

Comment:
```suggestion
        return query.selectFrom(item)
                .innerJoin(item.category, category).joinFetch()
                .where(
                        category.id.eq(categoryId),
```
위와 같이 어떤 조인인지 명시해주고, `category` 정보를 주로 많이 사용할 거같은데 fetch join하는게 network 자원 및 성능 개선에 좋아 보이네요

Reply:
왜 이부분은 수정 안되어있는데 reseolve를 했는지 알 수 있을까요??